pkg/healthcheck/healthcheck.go

```diff
@@ -1152,11 +1161,36 @@ func (hc *HealthChecker) allCategories() []category {
 						return &SkipError{Reason: "not run for non HA installs"}
 					},
 				},
+				{
+					description: "multiple replicas of control plane pods",
+					hintAnchor:  "l5d-control-plane-replicas",
+					warning:     true,
+					check: func(ctx context.Context) error {
+						if hc.isHA() {
+							return hc.controlPlaneComponentsAreHA()
+						}
+						return &SkipError{Reason: "not run for non HA installs"}
+					},
+				},
 			},
 		},
 	}
 }
 
+func (hc *HealthChecker) controlPlaneComponentsAreHA() error {
```

Comment:
I think this func name can be changes to reflect the fact that we are trying to establish a minimum number of available replicas.